daemon/src/maker_cfd_actor.rs

```diff
@@ -198,7 +198,8 @@ pub fn new(
                         );
 
                         current_contract_setup = Some(inbox.clone());
-                        for msg in contract_setup_message_buffer.clone() {
+
+                        for msg in contract_setup_message_buffer.drain(..) {
```

Comment:
`git absorb` would have likely picked the right commit here to squash this into :)

Reply:
😬